src/parser/metrics/CommentLines.ts

```diff
@@ -25,6 +30,16 @@ export class CommentLines implements Metric {
     }
 
     async calculate(parseFile: ParseFile, tree: Parser.Tree): Promise<MetricResult> {
+        switch (parseFile.language) {
+            case Languages.Python:
+                this.statementsSuperSet.push(
+                    new SimpleQueryStatement(
+                        "(expression_statement (string)) @python_multiline_comment"
+                    )
+                );
+                break;
+        }
```

Comment:
Let's don's use switches as long as we don't actually need them. An if is sufficient here.

Reply:
Replaced the switch-case with an if-condition